ios/LikeCoinApp/Info.plist

```diff
@@ -58,6 +64,10 @@
 			</dict>
 		</dict>
 	</dict>
+  <key>FacebookAppID</key>
+  <string>105087446989974</string>
+  <key>FacebookDisplayName</key>
+  <string>Liker</string>
```

Comment:
should change to Liker Land?

Reply:
the current name in our facebook app is `Liker`, not sure if they need to be the same
In fact I dont think this value shows up in our UX at all